apigw/src/enduser/app.ts

```diff
@@ -25,6 +25,7 @@ import session, {
 } from '../shared/session'
 import publicRoutes from './publicRoutes'
 import routes from './routes'
+import maproutes from './maproutes'
```

Comment:
Tiny nitpick: why not `map-routes` or `mapRoutes`? This is the third naming convention in the same import list